src/main/java/ca/yorku/cse2311/tab2pdf/ui/listener/SubtitleListener.java

```diff
@@ -0,0 +1,54 @@
+package ca.yorku.cse2311.tab2pdf.ui.listener;
+
+import ca.yorku.cse2311.tab2pdf.ui.MainJFrame;
+
+import java.awt.event.KeyEvent;
+import java.awt.event.KeyListener;
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Created by Varsha on 15-03-24.
+ */
+public class SubtitleListener implements KeyListener {
+
+    private final MainJFrame window;
+
+    /**
+     * @param window the window we are working with
+     */
+    public SubtitleListener(MainJFrame window) {
+
+        super();
+
+        this.window = window;
+    }
+
+    @Override
+    public void keyTyped(KeyEvent e) {
+        // do nothing
+    }
+
+    @Override
+    public void keyPressed(KeyEvent e) {
+        // do nothing
+    }
+
+    @Override
+    public void keyReleased(KeyEvent e) {
+        int line = 0;
```

Comment:
This should probably go in `keyPressed()` instead of this method. 

It appears this is always 1 update behind. When I type some extra characters to the end, the *Song Title* text field isn't updated right away (try adding two **z**'s at the end, only the first one will show up, until you make another change, which then allows the second **z** to show up)